packages/framer-motion/src/projection/node/create-projection-node.ts

```diff
@@ -55,6 +55,11 @@ const transformAxes = ["", "X", "Y", "Z"]
  */
 const animationTarget = 1000
 
+/**
+ * A mutable state containing the latest animation frame timestamp.
+ */
+const frameData = getFrameData()
```

Comment:
Why is this called data? Going at the comment I would say `frameState`

Reply:
Yeah this can also work